vmware_exporter/vmware_exporter.py

```diff
@@ -426,18 +448,29 @@ def vm_inventory(self):
     @run_once_property
     @defer.inlineCallbacks
     def datastore_labels(self):
-        datastores, dcs, folder_to_datacenter = yield parallelize(
+        datastores, dcs, folder_to_datacenter, folder_to_storagepod = yield parallelize(
             self.datastore_inventory,
             self.datacenter_inventory,
-            self.folder_to_datacenter
+            self.folder_to_datacenter,
+            self.folder_to_storagepod,
         )
 
         labels = {}
         for moid, datastore in datastores.items():
+            folder_id = datastore['parent']._moId
+
+            dc_name = ''
+            if folder_id in folder_to_datacenter:
+                dc_name = folder_to_datacenter[folder_id]['name']
+
+            ds_cluster = ''
+            if folder_id in folder_to_storagepod:
```

Comment:
If ds_cluster is blank then this test must be failing. @pryorda can you run this in your environment with a print here? Maybe the dict isn't properly populated.

Reply:
With the other print ^ then just a `print(folder_id, folder_to_storagepod)` here will be enough.